docker-compose.yml

```diff
@@ -57,3 +47,13 @@ services:
     networks:
       - opba-net
 
+  fintech-server:
+    environment:
+      - TPP_URL=http://open-banking-gateway:8085
+    build: fintech-examples/fintech-server
+    ports:
+      - "18086:8086"
+    entrypoint: "java -jar fintech-server-0.0.6-SNAPSHOT.jar"
```

Comment:
Suggest to find this jar name using some sh script (or simply `"java -jar fintech-server-*.jar`) - as soon as we use 0.0.7 it will break